romanesco/__init__.py

```diff
@@ -19,9 +19,13 @@
     'docker': executors.docker.run,
     'python': executors.python.run,
     'r': executors.r.run,
-    'workflow': executors.workflow.run
+    'workflow': executors.workflow.run,
+    'pyspark': executors.pyspark.run
```

Comment:
Yes,  that was my thinking. However, may be it should be 'spark.python', 'spark.r', 'spark.scala' ... ?